packages/manager/apps/web/client/app/domain/webhosting/order/steps/dnsConfiguration/domain-webhosting-order-steps-dnsConfiguration.html

```diff
@@ -29,7 +29,7 @@
             data-translate="domain_webhosting_order_dns_configuration_enable_hosting_help"
         ></span>
     </div>
-    <div>
+    <div data-ng-if="$ctrl.region !=='CA'">
```

Comment:
**nitpick:** We can use feature-flipping to handle these scenarios. 
Feel free to reach out to me in case you need more info on how to use feature-flipping.

Reply:
Hello Anoop, thanks for the review, I updated PR and used feature flipping (see feature-availability pull-request 322)